content/src/content/jcr_root/apps/core/wcm/components/teaser/v1/teaser/_cq_dialog/.content.xml

```diff
@@ -67,6 +67,19 @@
                                         jcr:primaryType="nt:unstructured"
                                         sling:resourceType="granite/ui/components/coral/foundation/container">
                                         <items jcr:primaryType="nt:unstructured">
+                                            <pretitleGroup
```

Comment:
@dymitrs - I don't think this needs to be grouped like the title or description, would remove the well wrapper here.

Reply:
Sure, I'll remove it